pages.zh/common/touch.md

```diff
@@ -0,0 +1,20 @@
+# touch
+
+> 创建文件并设置访问/修改时间。
+> 更多信息：<https://manned.org/man/freebsd-13.1/touch>。
+
+- 创建指定的文件：
+
+`touch {{路径/到/文件1 路径/到/文件2 ...}}`
+
+- 将文件的访问或修改时间设置为当前时间，如果文件不存在则不创建：
+
+`touch -c -{{a|m}} {{路径/到/文件1 路径/到/文件2 ...}}`
```

Comment:
IMO, -a, -m does not intuitively map to 訪問, 修改.
can we change the description to => 将文件的访问(a)或修改(m)时间设置为当前时....
Not sure if this align with the lint, but it will be more clear that "a" stands for 訪問, "m" stands for 修改